lib/rack_graphql/exception_json_response_middleware.rb

```diff
@@ -0,0 +1,42 @@
+module RackGraphql
+  class ExceptionJSONResponseMiddleware
+    DEFAULT_ERROR_STATUS_CODE = 500
+
+    def initialize(app, app_name:, error_status_code_map:, log_exception_backtrace:)
+      @app = app
+      @app_name = app_name
+      @error_status_code_map = error_status_code_map
+      @log_exception_backtrace = log_exception_backtrace
+    end
+
+    def call(env)
+      @app.call(env)
+    rescue AmbiguousParamError
```

Comment:
`AmbiguousParamError` is a custom exception representing a client error , not a system exception and should not be handled here. It's  [raised by the main middleware](https://github.com/RenoFi/rack-graphql/blob/f28ac5b88f852f9528edbe1ea98ade5a8d8e0a69/lib/rack_graphql/middleware.rb#L53-L70).